packages/roc-package-webpack-web-dev/src/config/roc.config.js

```diff
@@ -3,7 +3,8 @@ const config = {
         dev: {
             devMiddleware: {
                 noInfo: true,
-                quiet: false
+                quiet: false,
+                poll: false
```

Comment:
I'll update the PR with support for interval. 

And I'll take a look at aggregateTimout.